README.md

```diff
@@ -1,19 +1,164 @@
+	- Does the users account have Admin privileges? 
+		- -> True: 
+			- Has the user logged in recently?
+				- -> True: The Administrative user is automatically redirected to their admin harmonIQ profile page, where they have access to admin only features, such as the user list.
+				- -> False: The Administrative user is prompted by Spotify to sign in using their Spotify account credentials. Upon authentication the user is redirected to their Administrative harmonIQ profile.
+		- -> False: The user will have to reach out to the development team and apply for Admin privileges to be granted to their account.
+	
+**Profile Stories:**
+
+- User View, Top Tracks (Iteration 1) - A regular user would like to view their top Spotify tracks through harmonIQ.
+	- Does the user have a listening history with their Spotify account?
+		- -> True: Their top tracks will be pulled from Spotify and displayed on their harmonIQ profile.
+		- -> False: The top tracks section of their profile will be empty until they generate a listening history.
+
+***
+
+- _User View, Quiz Score_ (In Progress) - A regular user would like to view their harmonIQ quiz total score.
+	- Has the user played at least 1 quiz?
+		- -> True: The user may see their harmonIQ quiz total score by navigating to their profile, where it will be displayed among other statistics and information.
+		- -> False: The user will see that their harmonIQ quiz total score is defaulted to 0 until they have played at least 1 quiz.
+
+***
+
+- _Admin View, User List_ (In Progress) - A administrator would like to view the current active user list.
+	- Is there at least 1 other user on harmonIQ?
+		- -> True: The administrator may see all other active users, as well as other **admin only** information by navigating to their profile page.
+		- -> False: The user list will be empty and state that their are no active users on harmonIQ.
+
+
+**Quiz Stories:**
+
+- _Album Match Quiz_ (Iteration 1) - A user starts the Album Match Quiz, a game where they are given the artwork from a random album in their library and have to pick from a few choices which is the correct album name.
+	- -> The user answers incorrectly: The users score **does not** increase and the next question loads.
+	- -> The user answers correctly: The users score **does** increase, and the next question loads.
+	- -> The user answers the final question, either correctly or incorrectly: The user is brought to a results screen which shows how many questions they answered correctly.
+
+
+**Logout Stories**
+
+- _User Logout Process_ (Iteration 1) - A user is done currently using harmonIQ and wishes to logout from their profile.
+	- -> The user clicks the logout button: They are redirected to the harmonIQ login page and can no longer view their profile until the log back in.
+
+## Test Cases:
```

Comment:
Let's add the Test Cases to a Wiki page in the repo?! It'll be better to build upon them there and prevent the README from keep growing just with the test cases

Reply:
I'm not seeing the wiki header on the repo. Does @ChristianChidiac have to enable it? I've never worked with github wikis before so I might be missing something! Besides that, good idea, do you think it would make sense to split User Stories into a Wiki page as well since that is probably going to grow as well?

Reply:
I enabled the wikis

Reply:
Yeah, we can have a page with User Stories in there as well and add the link to both, User Stories and Test Cases, in the README for reference